mail-mta/sendmail/sendmail-8.16.1-r1.ebuild

```diff
@@ -0,0 +1,203 @@
+	dodir /var/spool/{mqueue,clientmqueue} /etc/conf.d
+
+	keepdir /var/spool/{clientmqueue,mqueue}
+
+	local emakeargs=(
+		DESTDIR="${D}" LIBDIR="/usr/$(get_libdir)"
+		MANROOT=/usr/share/man/man
+		SBINOWN=root SBINGRP=root UBINOWN=root UBINGRP=root
+		MANOWN=root MANGRP=root INCOWN=root INCGRP=root
+		LIBOWN=root LIBGRP=root GBINOWN=root GBINGRP=root
+		MSPQOWN=root CFOWN=root CFGRP=root
+	)
+
+	local dir
+	for dir in libsmutil sendmail mailstats praliases smrsh makemap vacation editmap; do
+		emake -j1 -C obj.*/${dir} "${emakeargs[@]}" install
+	done
+	for dir in rmail mail.local; do
+		emake -j1 -C obj.*/${dir} "${emakeargs[@]}" force-install
+	done
+
+	fowners root:smmsp /usr/sbin/sendmail
+	fperms 2555 /usr/sbin/sendmail
+	fowners smmsp:smmsp /var/spool/clientmqueue
+	fperms 770 /var/spool/clientmqueue
+	fperms 700 /var/spool/mqueue
+	dosym ../sbin/makemap /usr/bin/makemap
+	dodoc FAQ KNOWNBUGS README RELEASE_NOTES doc/op/op.ps
+
+	dodoc sendmail/{SECURITY,TUNING}
+	newdoc sendmail/README README.sendmail
+	newdoc smrsh/README README.smrsh
+
+	newdoc cf/README README.cf
+	newdoc cf/cf/README README.install-cf
+
+	dodoc -r contrib
+
+	cp -pPR cf/. "${ED}"/usr/share/sendmail-cf || die
+
+	insinto /etc/mail
+	if use mbox; then
+		newins "${FILESDIR}"/sendmail.mc-r1 sendmail.mc
+	else
+		newins "${FILESDIR}"/sendmail-procmail.mc sendmail.mc
+	fi
+
+	# See discussion on bug #730890
+	m4 "${ED}"/usr/share/sendmail-cf/m4/cf.m4 \
+		<(grep -v "${EPREFIX}"/usr/share/sendmail-cf/m4/cf.m4 "${ED}"/etc/mail/sendmail.mc) \
+		> "${ED}"/etc/mail/sendmail.cf || die "cf.m4 failed"
+
+	echo "include(\`/usr/share/sendmail-cf/m4/cf.m4')dnl" \
+		> "${ED}"/etc/mail/submit.mc || die "submit.mc echo failed"
+
+	cat "${ED}"/usr/share/sendmail-cf/cf/submit.mc \
+		>> "${ED}"/etc/mail/submit.mc || die "submit.mc cat failed"
+
+	echo "# local-host-names - include all aliases for your machine here" \
+		> "${ED}"/etc/mail/local-host-names || die "local-host-names echo failed"
+
+	cat <<- EOF > "${ED}"/etc/mail/trusted-users || die "trusted-users cat failed"
+		# trusted-users - users that can send mail as others without a warning
+		# apache, mailman, majordomo, uucp are good candidates
+	EOF
+
+	cat <<- EOF > "${ED}"/etc/mail/access || die "access cat failed"
+		# Check the /usr/share/doc/sendmail/README.cf file for a description
+		# of the format of this file. (search for access_db in that file)
+		# The /usr/share/doc/sendmail/README.cf is part of the sendmail-doc
+		# package.
+		#
+
+	EOF
+
+	cat <<- EOF > "${ED}"/etc/conf.d/sendmail || die "sendmail cat failed"
+		# Config file for /etc/init.d/sendmail
+		# add start-up options here
+		SENDMAIL_OPTS="-bd -q30m -L sm-mta" # default daemon mode
+		CLIENTMQUEUE_OPTS="-Ac -q30m -L sm-cm" # clientmqueue
+		KILL_OPTS="" # add -9/-15/your favorite evil SIG level here
+
+	EOF
+
+	if use sasl; then
+		dodir /etc/sasl2
+		cat <<- EOF > "${ED}"/etc/sasl2/Sendmail.conf || die "Sendmail.conf cat ailed"
```

Comment:
'Sendmail' vs sendmail looks odd, but if it's supposed to be that way, fair enough.

Reply:
I agree, but I'd rather leave it how it was without looking deeper into it.